nested/db-postgres.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.